src/MagnumPlugins/KtxImporter/formatMapping.py

```diff
@@ -0,0 +1,138 @@
+
+with open(format_header, encoding='utf-8') as f:
+    lines = f.readlines()
+    for line in lines:
+        # Get mapping from VkFormat to Magnum::Vk::PixelFormat
+        # PixelFormat and Vk::PixelFormat names are identical
+        match = re.search('^\s+(Compressed)?(\w+) = VK_FORMAT_(\w+),?$', line)
+        if match:
+            compressed = match.group(1) != None
+            magnum_name = match.group(2)
+            vulkan_name = match.group(3)
+            assert(vulkan_name in vulkan_formats)
+
+            type = re.search('\w+_([U|S](NORM|INT|FLOAT|RGB))\w*', vulkan_name)
+            assert type != None
+            assert type.group(1) != 'URGB'
+
+            formats.append(Format(compressed, magnum_name, vulkan_formats[vulkan_name], type.group(1)))
+
+if len(formats) != 135:
+    print('Unexpected number of formats')
+
+# https://docs.python.org/dev/library/itertools.html#itertools-recipes
+def partition(pred, iterable):
+    t1, t2 = itertools.tee(iterable)
+    return itertools.filterfalse(pred, t1), filter(pred, t2)
+
+# There's no PVRTC2 in Magnum::PixelFormat
+formats = [f for f in formats if not f.magnum.startswith('Pvrtc2')]
+
+header = '''/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+                2020, 2021 Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2021 Pablo Escobar <mail@rvrs.in>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/* Autogenerated from formatMapping.py! Do not edit! */
+
+'''
+
+with open(file_out, 'w', encoding='utf-8') as outfile:
+    compressed = lambda f : f.compressed
+    formats, compressed_formats = partition(compressed, formats)
+
+    outfile.write(header)
+
+    outfile.write('#ifdef _p /* PixelFormat */\n')
+    for format in formats:
+        outfile.write('_p(' + format.vulkan + ', ' + format.magnum + ', ' + format.type + ')\n')
```

Comment:
To avoid accidents I think it could be useful to store the enum names as a comment? Like, if you'd save the `vulkan_name` in the above loop as well and then

```suggestion
        outfile.write('_p({}, {}, {}) /* {} */\n'.format(format.vulkan, format.magnum, format.type, format.vulkan_name)
```

Same for the compressed formats.